pom.xml

```diff
@@ -272,18 +272,18 @@
             </dependency>
             <dependency>
                 <groupId>org.mockito</groupId>
-                <artifactId>mockito-testng</artifactId>
-                <version>0.4.31</version>
+                <artifactId>mockito-junit-jupiter</artifactId>
+                <version>5.0.0</version>
```

Comment:
We need `4.11.0`, `5.x` require jdk 11 at least

Reply:
Updated